packages/backend/src/netcat.js

```diff
@@ -125,11 +123,19 @@ const inspectPlayersDummy = async () => playersWithDetailsDummy
 const inspectPlayers = NC_DISABLE === 'true' ? inspectPlayersDummy : inspectPlayersReal
 
 const whoIsPlayingReal = async (serverConfig) => {
-  const playerList = [...await nc(serverConfig, [
+  const playerData = await nc(serverConfig, [
     'RefreshList',
-  ])][0].PlayerList
+  ])
+
+  if (playerData) {
+    const playerList = [...await nc(serverConfig, [
+      'RefreshList',
+    ])][0].PlayerList
```

Comment:
Would `const playerList = [...playerData][0].PlayerList` work?

Reply:
Yes it would, updating that now.